tests/gdtest/gdtest.h

```diff
@@ -63,7 +63,7 @@ int _gdTestErrorMsg(const char* file, unsigned int line, const char* string, ...
 
 /* public assert functions */
 #define gdAssertImageEqualsToFile(ex,ac) gdTestImageCompareToFile(__FILE__,__LINE__,NULL,(ex),(ac))
-#define gdAssertImageFileEqualsMsg(ex,ac,ms) gdTestImageCompareFiles(__FILE__,__LINE__,(ms),(ex),(ac))
+#define gdAssertImageFileEqualsMsg(ex,ac,ms) gdTestImageCompareToFile(__FILE__,__LINE__,(ms),(ex),(ac))
```

Comment:
gdAssertImageFileEqualsMsg appears to be unused.  if it's broken, lets just delete it.